src/atoms/hooks/useSignalRMessages.ts

```diff
@@ -12,7 +12,7 @@ import { environment } from 'src/atoms/utils/auth_environment';
 
 /* c8 ignore start */
 /**
- * @deprecated Being deprecated from amplify-components-library, use the new @equinor/subsurface-app-management package instead
+ * @deprecated Being deprecated from amplify-component-lib-library, use the new @equinor/subsurface-app-management package instead
```

Comment:
On all the deprecated comments it already said -library, so now it became component-lib-library 😬

Reply:
Pushed a commit now